desktop/src/main.js

```diff
@@ -14,7 +14,7 @@ import path from 'path';
 import { handleSquirrelEvent } from './handle-squirrel-event';
 
 const isDev = process.env.ENV === 'dev';
-const url = isDev ? 'http://localhost/comm/' : 'https://web.comm.app';
+const url = isDev ? 'http://localhost:3000/comm/' : 'https://web.comm.app';
```

Comment:
I think this is a good change given the recent changes in the dev environment to no longer use Apache as a proxy.

@MichalGniadek, does this change make sense to you?

Reply:
Yeah, definitely!